Simplified/NYPLAnnotations.swift

```diff
@@ -142,7 +142,7 @@ final class NYPLAnnotations: NSObject {
           debugPrint(#file, "Posted Last-Read \(((parameters["target"] as! [String:Any])["selector"] as! [String:Any])["value"] as! String)")
         }
       } else {
-        guard let error = error as? NSError else { return }
+        guard let error = error as NSError? else { return }
```

Comment:
This is not directly related to your change per se, but the sum of this code does not seem right to me:

1. The type of response is always `HTTPURLResponse` and can be safely cast using `as!`.

2. If the type of the response is `HTTPURLResponse` and the status code is not 200, we silently ignore it. Not at least logging an error here seems wrong to me, but I think we're actually supposed to be retrying later.

3. The logging code when the response is 200 will crash the app if undocumented keys are not present.

4. The call to `addToOfflineQueue` on line 147 will never be hit because response is always of type `HTTPURLResponse`.